nntrainer/tensor/float_tensor.h

```diff
@@ -35,6 +35,35 @@ class FloatTensor : public TensorBase {
    */
   FloatTensor(std::string name_ = "", Tformat fm = Tformat::NCHW);
 
+  /**
+   * @brief Construct a new FloatTensor object
+   *
+   * @param d
```

Comment:
Add a short description for param with in/out.

Reply:
Fixed